ephios/static/ephios/js/event_notifications.js

```diff
@@ -0,0 +1,9 @@
+$(document).ready(function () {
+    $("input:radio[name ='action']").change(function (event) {
+        if ($("input:radio[name ='action']:checked").val() === "participants")
+            $("#div_id_mail_content").slideDown();
+        else
+            $("#div_id_mail_content").slideUp();
+    });
+
```

Comment:
I'm not a big fan of adding this `show-if-this-is-checked` logic in multiple views, we could add something like this as a general purpose utility: 
https://github.com/pretix/pretix/blob/a897f60fc5a22d9eb9bf4107111eb8ea05d09b3b/src/pretix/static/pretixcontrol/js/ui/main.js#L287